### ISSUE TITLE
Error during start default cardinal server on Ubuntu in Windows 11 WSL

### ISSUE THREAD

**Issue by k-karuna** [Argus-Labs/world-engine#743]
Hi all!
I'm facing error during starting default server running `world cardinal dev`:

```
Starting Cardinal...
# pkg.world.dev/world-engine/cardinal/server
../../go/pkg/mod/pkg.world.dev/world-engine/cardinal@v1.2.5-beta/server/server.go:175:29: undefined: handler.PostCQL
```

I'm using Ununtu 22.04 on Windows 11 WSL.
world version World CLI 1.2.7
go version go1.22.2 linux/amd64

I'd like to point out that same steps works fine without error on standalone Ubuntu 22.04 (not on WSL).

**Comment by zulkhair** [comment]
Hi @k-karuna,

Have you previously run `world cardinal dev` using another version of World CLI?

If so, maybe it's because of the Go Mod cache. Try to run this method (run inside the `{you-game-shard-dir}/cardinal` directory)
1. Clear the Module Cache: Sometimes, the cache might be corrupted or contain outdated information. You can try clearing the module cache by running `go clean -modcache`
2. Update Dependencies: Ensure that your dependencies are up-to-date. Run `go get -u ./...` This will update all the dependencies to their latest versions.

**Comment by smsunarto** [comment]
Hey @k-karuna, just checking in. Is this still an issue?

**Comment by k-karuna** [comment]
@zulkhair Hi, sorry for long reply, was on vacation

Tried from scratch on clean Ubuntu 24.04, same thing
After `go clean -modcache` and `go get -u ./...` i'm receiving next output:

Running Cardinal in dev mode
Press Ctrl+C to stop

```
Redis: localhost:6379
Cardinal: localhost:4040
Cardinal Editor: localhost:3000
Preparing Cardinal Editor...

8bdccdb3c190ce29eb0e56ad7cdf0988c2f1300eecee9333e1f132752d97d4aa
Starting Cardinal...
# pkg.world.dev/world-engine/cardinal/router/iterator
../../go/pkg/mod/pkg.world.dev/world-engine/cardinal@v1.3.2/router/iterator/iterator.go:133:3: unknown field Body in struct literal of type sign.Transaction
# pkg.world.dev/world-engine/cardinal/server/handler
../../go/pkg/mod/pkg.world.dev/world-engine/cardinal@v1.3.2/server/handler/tx.go:64:33: tx.Body undefined (type *sign.Transaction has no field or method Body)
```

**Comment by k-karuna** [comment]
BTW I tried to clone world engine on Windows (not on WSL) with latest commit https://github.com/Argus-Labs/world-engine/commit/8ad754dec16e084fad40db77b253d45cea8dc3d8

Looks like `go build` in cardinal direcory leads to the same issue:

```
go build
# pkg.world.dev/world-engine/cardinal/server
server\server.go:175:29: undefined: handler.PostCQL
```

go version go1.22.2 windows/amd64

**Comment by mismirnov** [comment]
Hi @zulkhair @smsunarto
I'm facing the same error during starting default server running `world cardinal start`:

I'm using MacOS Ventura 13.0.1
world version World CLI 1.2.8
go version go1.22.2 darwin/arm64

<img width="1167" alt="docker_container" src="https://github.com/Argus-Labs/world-engine/assets/58223117/95a22dde-1f8e-4591-acc0-650774439681">

<img width="1158" alt="nakama_error" src="https://github.com/Argus-Labs/world-engine/assets/58223117/b68b8f3e-80f8-4d27-9bc6-b8e1da099509">

**Comment by smsunarto** [comment]
@mismirnov that is a different error, but the solution both of you guys here should be the same. We just updated the starter-game-template with a new version of Cardinal. Can you try doing `world create` again so it gives you the new version?

**Comment by mismirnov** [comment]
@smsunarto Yes, ofc.

**Comment by mismirnov** [comment]
@smsunarto Issue resolved.
Thank you.

**Comment by smsunarto** [comment]
Closing this as resolved. Let me know if you still have issues @k-karuna

**Comment by k-karuna** [comment]
@smsunarto unfortunately my issue didn't resolved on newest cardinal version 1.3.2, the problem remains the same.

```
world cardinal dev
╭────────────────────────────────────────╮
│                Cardinal                │
╰────────────────────────────────────────╯
Running Cardinal in dev mode
Press Ctrl+C to stop

Redis: localhost:6379
Cardinal: localhost:4040
Cardinal Editor: localhost:3000
Preparing Cardinal Editor...

0f8281d61bb5d4c5ed6a2e56d37016d9420cdc155420de9867f6a52cec8e8ec1
Starting Cardinal...
# pkg.world.dev/world-engine/cardinal/server
../../go/pkg/mod/pkg.world.dev/world-engine/cardinal@v1.3.2/server/server.go:175:29: undefined: handler.PostCQL
```
go version go1.22.1 linux/amd64
Also tried on go1.22.2 and go1.22.3 versions

Executing `go build` in `{you-game-shard-dir}/cardinal` dierctory results in same output

But `world cardinal start` works

As @zulkhair mentioned https://github.com/Argus-Labs/world-engine/issues/743#issuecomment-2070626572, i've tried to execute as i wrote above https://github.com/Argus-Labs/world-engine/issues/743#issuecomment-2115956987
`go clean -modcache` and `go get -u ./...` and it results in

```
world cardinal dev
╭────────────────────────────────────────╮
│                Cardinal                │
╰────────────────────────────────────────╯
Running Cardinal in dev mode
Press Ctrl+C to stop

Redis: localhost:6379
Cardinal: localhost:4040
Cardinal Editor: localhost:3000
Preparing Cardinal Editor...

1c808d9c505c08615c59845458e69fbf693008775849f6e5f3554eb6ccbcb0fe
Starting Cardinal...
# pkg.world.dev/world-engine/cardinal/router/iterator
vendor/pkg.world.dev/world-engine/cardinal/router/iterator/iterator.go:133:3: unknown field Body in struct literal of type sign.Transaction
# pkg.world.dev/world-engine/cardinal/server/handler
vendor/pkg.world.dev/world-engine/cardinal/server/handler/tx.go:64:33: tx.Body undefined (type *sign.Transaction has no field or method Body)
```
